Object Containers/src/main/Person.java

```diff
@@ -0,0 +1,58 @@
+package main;
+
+import java.util.Comparator;
+import java.util.Objects;
+
+public class Person implements Comparator {
```

Comment:
Comparatorul ar fi trebuit implementat intr-o clasa disticnta care sa contina doar implementarea metodei `compareTo`. Am vazut ca ai facut asta in clasa `AgeSorted`, clasa asta nu ar fi trebuit sa implementeze interfata `Comparator`.